src/Microsoft.AspNet.Razor/Generator/TagHelpers/TagHelperPrefixDirectiveCodeGenerator.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Parser.SyntaxTree;
+
+namespace Microsoft.AspNet.Razor.Generator
+{
+    /// <summary>
+    /// A <see cref="SpanCodeGenerator"/> responsible for generating 
+    /// <see cref="Compiler.TagHelperPrefixDirectiveChunk"/>s.
+    /// </summary>
+    public class TagHelperPrefixDirectiveCodeGenerator : SpanCodeGenerator
+    {
+        /// <summary>
+        /// Instantiates a new <see cref="TagHelperPrefixDirectiveCodeGenerator"/>.
+        /// </summary>
+        /// <param name="prefix">
+        /// Text used as <see cref="TagHelpers.TagHelperDescriptor"/>'s 
+        /// <see cref="TagHelpers.TagHelperDescriptor.Prefix"/>.
```

Comment:
use your "Text used as a required prefix when matching HTML ..." wording in all relevant doc comments